src/plugins/timelist/Timelist.vue

```diff
@@ -197,21 +197,22 @@ export default {
                 this.listActivities();
             }
         },
-        updateTimestamp(_bounds, isTick) {
+        updateTimestamp(bounds, isTick) {
             if (isTick === true && this.openmct.time.clock() !== undefined) {
                 this.updateTimeStampAndListActivities(this.openmct.time.clock().currentValue());
+            } else if (isTick === false && this.openmct.time.clock() === undefined) {
+                // set the start time for fixed time using the selected bounds start
+                this.updateTimeStampAndListActivities(bounds.start);
             }
         },
         setViewFromClock(newClock) {
             this.filterValue = this.domainObject.configuration.filter;
-            const isFixedTime = newClock === undefined;
-            if (isFixedTime) {
+            this.isRealTime = newClock !== undefined;
```

Comment:
Let's remove this change if it's the same as before and keep `this.isFixedTime`

Reply:
Inverted back to fixed time instead of real time.